src/components/FeatureViews/AdminPortal/ActionLog.js

```diff
@@ -113,57 +126,41 @@ const ActionLog = () => {
             sort: currentSort.sort,
             objectType: objectType.toLowerCase(),
             userId: currentId.toString(),
-        },
-        "onCompleted": (data) => {
-            setLogData(data);
-        },
+        }
     });
     const [userOptions, setUserOptions] = useState(null)
     const adminOptions = ["Owner", "Receptionist", "Assistant"];
     const actionOptions = ["Add", "Edit", "Delete"]
     const objectOptions = ["Student", "Admin", "Parent", "Instructor", "Payment", "Registration", "Tutoring", "Course", "Discount", "Price Rules"]
 
-    useEffect(() => {
-        logs()
-        fetchAdmins()
-    }, [])
-
-    const renderChip = (action) => {
-        switch (action) {
-            case "Edit":
-                return (<Chip style={{ borderRadius: 3, width: 70, backgroundColor: "#FFDD59" }} label="Edit"></Chip>)
-            case "Add":
-                return (<Chip style={{ borderRadius: 3, width: 70, backgroundColor: "#78E08F" }} label="Add"></Chip>)
-            case "Delete":
-                return (<Chip style={{ borderRadius: 3, width: 70, backgroundColor: "#FF7675" }} label="Delete"></Chip>)
-        }
-    }
+    if (loading || admins.loading) return <Loading />;
+    // if (error || admins.error) return "error";
+    // for some reason returning error will break on filtering for objects. Not sure why, maybe it's a backend issue
```

Comment:
You need to return a div/component so something like `return <div>{error.message}</div>`

Reply:
this still doesn't fix the error

Reply:
this only occurs for price rules as well